receiptui/src/main/java/com/hyperwallet/android/ui/receipt/view/ReceiptDetailFragment.java

```diff
@@ -118,16 +119,14 @@ private void setTransactionView(@NonNull final Receipt receipt, @NonNull final V
         if (CREDIT.equals(receipt.getEntry())) {
             transactionAmount.setTextColor(transactionAmount.getContext()
                     .getResources().getColor(R.color.positiveColor));
-            transactionAmount.setText(transactionAmount.getContext()
-                    .getString(R.string.credit_sign, currencyString, receipt.getAmount()));
+            transactionAmount.setText(Utils.formatCurrency(receipt.getCurrency(), receipt.getAmount()));
             transactionTypeIcon.setTextColor(transactionTypeIcon.getContext()
                     .getResources().getColor(R.color.positiveColor));
             transactionTypeIcon.setText(transactionTypeIcon.getContext().getText(R.string.credit));
         } else if (DEBIT.equals(receipt.getEntry())) {
             transactionAmount.setTextColor(transactionAmount.getContext()
                     .getResources().getColor(R.color.negativeColor));
-            transactionAmount.setText(transactionAmount.getContext()
-                    .getString(R.string.debit_sign, currencyString, receipt.getAmount()));
+            transactionAmount.setText("-" + Utils.formatCurrency(receipt.getCurrency(), receipt.getAmount()));
```

Comment:
why set `"-"`

Reply:
Discussed with @malaw-hw, we should use `R.string.debit_sign` instead of `"-"`

Reply:
Review comments implemented